docs/enriching-your-data/available-enrichments/custom-javascript-enrichment/index.md

```diff
@@ -131,6 +139,117 @@ Manipulating your event stream with Javascript is powerful, but can be dangerous
 - include CPU-intensive tasks without being aware of the impact on your event processing time
 - allow untrusted parties to write your script – the script has access to the Java standard library and therefore to your filesystem.
 
+## Testing your enrichment
+
+You can test your enrichment with [Snowplow Micro](/docs/getting-started-with-micro/what-is-micro/index.md) before adding it to your production pipeline. Follow the Micro [usage guide](/docs/getting-started-with-micro/configuring-enrichments/index.md) to set up Micro and configure it to use your enrichment.
+
+:::tip Testing `.js` files directly
+
+One trick up Micro’s sleeve is that while you can add the full enrichment configuration in a JSON file (with your JavaScript encoded in base64), you can also just drop a `.js` file in your enrichments folder. Micro will interpret it as JavaScript enrichment code. This way, you don’t need to keep converting your enrichment to base64 as you iterate on it. _(Make sure to either use a `.json` or a `.js` file, but not both.)_
+
+:::
+
+Let’s walk through an example.
+
+### Step 1. Write your enrichment
+
+For instance, to add an avocado to all your events:
```

Comment:
```suggestion
For instance, to add an avocado context entity to all your events:
```

Reply:
I changed to avocados in plural since it says “a lot”. I can add “context entity” but that would dial down the playfulness of the example...

Reply:
I also like that we don't know which one to use, so we use both...

Reply:
That's fair! My point was just that when someone comes across this section of the docs without previously being exposed to what a JS enrichment does, they probably won't understand what is happening here – the section doesn't explain that the goal is to add context entities (for instance, one may think that we are creating new avocado events). But that's quite a small nitpick.

Reply:
Presumably they would’ve read the rest of the page above this section, which explains this in detail!

Reply:
In any case, we need to rewrite the whole JS enrichment page because it’s very confusing. We’ll tackle that in a separate review